app/serializers/student_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class StudentSerializer
+  include FastJsonapi::ObjectSerializer
+    attributes :id,
```

Comment:
Thanks for taking the lead with designing Student attributes. To Do: to communicate this addition to the FE. I have added a tab to the spreadsheet to document it: https://docs.google.com/spreadsheets/d/1yoe0zdGWziHRCtJGQhfjDmUXlCQcM6ei1qt6v4WDIHc/edit#gid=672441840